src/modules/previewpane/common/handlers/PreviewHandlerBase.cs

```diff
@@ -201,7 +201,7 @@ private static void RegisterPreviewHandler(string name, string extensions, strin
             }
 
             // Add preview handler to preview handler list
-            using (RegistryKey handlersKey = Registry.LocalMachine.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))
+            using (RegistryKey handlersKey = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))

```

Comment:
Any reason for switching from HKLM to HKCR?

Reply:
It is better to install it on per user basis than across the system for all users.